### PR TITLE
Update merchant invoice status

### PR DESCRIPTION
**Pull request by B-gann21**
- set an alias for MerchantInvoices show page
- create an update method for MerchantInvoiceController
- add a form_with to build the select box for item status (query params are used to pass the item_id to the controller's update method)

### CODE REVIEW

File: app/views/merchant_invoices/show.html.erb [hunk 1]
```diff
@@ -33,6 +33,12 @@
         <p>Status: <%= @invoice.get_invoice_item(item.id).status %></p>
       </li>
     </ul>
+    <%= form_with url: "/merchants/#{@invoice.get_invoice_item(item.id).item.merchant_id}/invoices/#{@invoice.id}?item_id=#{item.id}", method: :patch, local: true do |form| %>
+      <%= form.label :select_status %>
+      <%= form.select :select_status,
+                      ['pending', 'packaged', 'shipped'] %>
+      <%= form.submit 'Update Item Status' %>
+    <% end %>
```
Comment (amsalmeron) [inline app/views/merchant_invoices/show.html.erb:42]: This select form look nicceeeeee. I like the setup. 
Does the user story want to have the current status selected by default? If so, that would be my only recommendation.

Comment (B-gann21) [reply app/views/merchant_invoices/show.html.erb:42]: Good catch! I'll add that now and push it up

Comment (amsalmeron) [reply app/views/merchant_invoices/show.html.erb:42]: Refactor looking good. Ready for merge.

**Comment by Eagerlearn** [comment]
* Brylan the code looks good and is ready to merge, please see my question below
* Question: when you set the alias to 'invoice' were you planning on using the form with model?

**Comment by B-gann21** [comment]
Good catch - I tried to use a form with model at first, which did require the alias. However, when it came that I needed to use query params for that form, I went back to just using form with url

**Comment by CLRM1** [comment]
Great work! Tests passing and user story flow with local host also looking good.

**Comment by CLRM1** [comment]
Great work! Tests passing and user story flow with local host also looking good.

**Pull request merged**